src/main/java/com/launchacademy/reviews/controllers/BerriesApiV1Controller.java

```diff
@@ -86,18 +86,20 @@ public ResponseEntity<Map<String, Berry>> editBerry(@PathVariable Long id) {
   }
 
   @PutMapping("/{id}/edit")
-  public ResponseEntity<Map<String, Berry>> updateBerry(@PathVariable("id") long id, @RequestBody Berry berryEdits) {
-      Optional<Berry> berryData = berryService.findById(id);
-      Map<String, Berry> dataMap = new HashMap<>();
-      if(berryData.isPresent()) {
-        Berry oldBerry = berryData.get();
-        Berry editedBerry = berryService.updateBerry(oldBerry, berryEdits);
-        dataMap.put("berry", editedBerry);
-        return new ResponseEntity<>(dataMap, HttpStatus.OK);
+  public ResponseEntity<Map<String, Berry>> updateBerry(@PathVariable("id") long id,
+      @RequestBody Berry berryEdits) {
```

Comment:
When we edit an entity, we want to make sure that the same validations are performed. Add BindingResult and validate just like when creating.